features/FxA-108-update-deps/README.md

```diff
@@ -0,0 +1,50 @@
+# FxA-108: update dependencies in FxA projects
+
+> https://github.com/mozilla/fxa-features/issues/17
+
+## Problem Summary
+
+This feature milestone deals with the following problems:
```

Comment:
Your metrics section hits on a unifying theme that I think it's worth calling out explicitly here: it's hard for us to respond to security issues in our dependencies in a timely manner.  The four problems you list below are all contributing factors to this, but the reason we care about them is because of the security implications.